### PR TITLE
Product filters

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**

### CODE REVIEW

File: src/Context/Filter_context.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const useFilter = () => useContext(FilterContext)
+
+const FilterProvider = ({children}) => {
+
+    const [state,dispatch] = useReducer(filterReducerFunc,{
+        sorting: null,rating:null, categories:{fiction: false,
+            spiritual: false, biography:false, horror: false},
+            price:500
+        });
```
Comment (Pratik1005) [inline src/Context/Filter_context.js:14]: just a suggestion: can use variable for initial state for code readability

**Review by Pratik1005 — APPROVED** [review]
Other than suggestion, good to go

File: src/pages/ProductsPage/Products.jsx [hunk 2]
```diff
@@ -95,12 +111,45 @@ const Products = () => {
...
+         
+          return (
+            <div>
+            <div class="products-card-container">
+            <div class="product-card">
+                <div class="badge">{rating}</div>
+                <div class="product-tumb">
+                    <img src={img}
+                        alt=""/>
+                </div>
```
Comment (sushmita12345) [inline src/pages/ProductsPage/Products.jsx]: Suggestion: add alt attribute

File: src/Context/Filter_context.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { createContext, useContext, useReducer } from "react";
+import { filterReducerFunc } from "../reducers/FilterReducer";
+
+const FilterContext = createContext()
+
+const useFilter = () => useContext(FilterContext)
+
+const FilterProvider = ({children}) => {
+
+    const [state,dispatch] = useReducer(filterReducerFunc,{
```
Comment (sushmita12345) [inline src/Context/Filter_context.js:10]: Suggestion: you can change the variable name of state, dispatch for better code readability when you add more Context Provider

**Review by sushmita12345 — APPROVED** [review]
Clean code, no breakpoint.
You can move ahead to merge.

File: src/Context/Filter_context.js (hunk 3 above)
Comment (Mandhularajitha) [reply src/Context/Filter_context.js:10]: Change Variable name of state

**Review by Mandhularajitha — APPROVED** [review]

File: src/Context/Filter_context.js (hunk 3 above)
Comment (vaishnavipaliya11) [reply src/Context/Filter_context.js:10]: But it might unmark the coding pattern

**Pull request merged**